failed to go build -tags="appengine"

env:
go 1.16.4
Windows 10
use example code from Quick start section

```
go build -tags="appengine"
# github.com/ugorji/go/codec
..\..\go\pkg\mod\github.com\ugorji\go\codec@v1.1.7\encode.go:651:9: undefined: mapIter
..\..\go\pkg\mod\github.com\ugorji\go\codec@v1.1.7\encode.go:652:2: undefined: mapRange
```

if upgrade codec to latest version(1.2.6), there is no error will be return.

--- a/go.mod
+++ b/go.mod
@@ -5,10 +5,11 @@ go 1.13
 require (
 	github.com/gin-contrib/sse v0.1.0
 	github.com/go-playground/validator/v10 v10.4.1
+	github.com/goccy/go-json v0.5.1
 	github.com/golang/protobuf v1.3.3
 	github.com/json-iterator/go v1.1.9
 	github.com/mattn/go-isatty v0.0.12
 	github.com/stretchr/testify v1.4.0
-	github.com/ugorji/go/codec v1.1.7
+	github.com/ugorji/go/codec v1.2.6
 	gopkg.in/yaml.v2 v2.2.8
 )

